utils/static-device-names/files/static-device-names.init

```diff
@@ -0,0 +1,359 @@
+#!/bin/sh /etc/rc.common
+
+# This script is part of the static-device-names package.
+# See /usr/share/doc/static-device-names
+#
+# Copyright (C) 2024 qualIP Software.
+# Released under the GPL v3.0 or later.
+#
+# Location: /etc/init.d/static-device-names
+
+START=11
+NAME=static-device-names
+TRACE=${TRACE:-0}
+
+# don't run within buildroot
+[ -n "${IPKG_INSTROOT}" ] && return 0
+
+###################
+# LOCAL FUNCTIONS #
+###################
+
+N=$'\n'  # New line
+
+LOG() {
+  local level ; level=$1 ; shift
```

Comment:
You should use a standardized variable definitions header in all your functions. The source code is read more often than it is written, so a standardized format should be selected.

```
function() {
local var1="$1"
local var2="$2"

local var3

...
}
```

Reply:
That *is* my standard ;-)
Using `local` or `export` separately from assignment is a habit because the assignment masks return values of commands executed, such as in `local v=$(false)`. For straight assignments, like `local v=$1`, it indeed does not matter.

I'll adapt the code to your liking, no problem. If you have a specific style guide to follow, I can do that too.